evaluation.py

```diff
@@ -1,45 +1,26 @@
 import tensorflow as tf
-from karas.metrics import MeanIoU
-from tensorflow.metrics import F1Score
-from karas.metrics import AUC
 from tensorflow.keras.callbacks import TensorBoard
 
-# before starting:
-# 0a. predict testing images
-# 0b. save newly predicted test images
-# 0c. read ground truth image
+# Load saved models
+unet_reconstructed = keras.models.load_model('/content/unet_model_h5.h5')

```

Comment:
I don't know if this will work locally because its the path for google drive. we should either save the models in the project and call those, or decide something else to do